scripts/create_newcase

```diff
@@ -188,12 +201,11 @@ sub _create_caseroot_tools
 }
 
 
-
 sub _check_input_options
 {
     my ($Case,$opts) = @_;
     
-    # Values in uppercase here have a direct correspondence to Entries in the case XML files
+    # Values in uppercase here have a direct correspondence to Entries in the caseroot XML files
```

Comment:
I don't understand this change?

Reply:
I think you mentioned the wrong "rocky". I am not rdunlap.